Cargo.toml

```diff
@@ -48,6 +48,8 @@ exclude = [
 resolver = "2"
 
 [patch.crates-io]
+aws-nitro-enclaves-cose_0_1 = { git = "https://github.com/fortanix/aws-nitro-enclaves-cose.git", branch = "raoul/0.1.1", package = "aws-nitro-enclaves-cose" }
```

Comment:
Can we have a comment here on why there are two versions of the same dependency?

Reply:
One patches `aws-nitro-enclaves-cose` version 0.1, the other 0.5. Both apply a similar fix. I was hoping this was clear from the name. What kind of comment would you like?